### PR TITLE
improve code to support rhel

### PR DESCRIPTION
**Pull request by linup2011**

### CODE REVIEW

**Comment by dceara** [comment]
This depends on https://github.com/ovn-org/ovn-fake-multinode/pull/29 so I'll wait with review until the ovn-fake-multinode PR gets merged.

File: ovn-fake-multinode-utils/playbooks/install-dependencies.yml [hunk 1]
```diff
@@ -2,13 +2,7 @@
...
-        name:
-          - git
-          - docker
-          - docker-distribution
-          - gcc
-          - openvswitch
-          - python3-pyyaml
-          - python3-virtualenv
-          - python3-devel
+      shell: yum install git  gcc openvswitch python3-pyyaml python3-virtualenv python3-devel  --skip-broken -y
```
Comment (dceara) [inline ovn-fake-multinode-utils/playbooks/install-dependencies.yml:5]: Nit: extra whitespace between `git` and `gcc`.

File: ovn-fake-multinode-utils/playbooks/install-dependencies.yml [hunk 2]
```diff
@@ -2,13 +2,7 @@
...
+    - name: Install container command
+      shell: yum install docker docker-distribution -y ||  yum install podman podman-docker -y
```
Comment (dceara) [inline ovn-fake-multinode-utils/playbooks/install-dependencies.yml:7]: Nit: extra whitespace before `yum install podman podman-docker -y`.

File: do.sh [hunk 3]
```diff
@@ -163,13 +198,52 @@ function install_ovn_fake_multinode() {
 
     # Clone repo locally
     clone_component ovn-fake-multinode ${ovn_fmn_repo} ${ovn_fmn_branch} || rebuild_needed=1
-    clone_component ovs ${ovs_repo} ${ovs_branch} || rebuild_needed=1
-    clone_component ovn ${ovn_repo} ${ovn_branch} || rebuild_needed=1
+
+    if [ -n "$RPM_OVS" ]
```
Comment (dceara) [inline do.sh:202]: Do you mind adding a section in the readme about these new variables that allow deployments with prebuilt RPMs? I.e., documenting `RPM_OVS`, `RPM_OVN_COMMON`, `RPM_OVN_CENTRAL`, `RPM_OVN_HOST`, `RPM_SELINUX`.

File: do.sh [hunk 4]
```diff
@@ -45,14 +48,46 @@ function install_deps() {
...
+    echo "-- Configuring podman local registry on tester nodes"
+    if which podman
+    then
+        for container_name in `podman ps | grep -v "CONTAINER ID" | awk '{print $1}'`
+        do
+            podman stop $container_name
+            podman rm $container_name
+        done
+        [ -d /var/lib/registry ] || mkdir /var/lib/registry -p
+        podman run --privileged -d --name registry -p 5000:5000 \
```
Comment (dceara) [inline do.sh:72]: Shouldn't this part be moved to `install_deps()` where we install and start local dependencies? Also, I don't think we need `registries = ['localhost:5000']` on the tester node, right?

Comment (linup2011) [reply do.sh:72]: I added this in case someone use the same system as both tester node and worker node

File: do.sh (hunk 3 above)
Comment (linup2011) [reply do.sh:202]: sure, I will

File: do.sh (hunk 4 above)
Comment (linup2011) [reply do.sh:72]: and this is used to configure podman, so added it here. should I remove it to install_deps?

Comment (dceara) [reply do.sh:72]: Yes, I think everything that's related to the "tester" node should go to `install_deps()` in the "-- Installing local dependencies" section.

**Comment by linup2011** [comment]
updated

**Pull request merged**